build/lib/helm_require.sh

```diff
@@ -45,15 +45,22 @@ spec:
 REQUIRES_CONFIG_FILE=helm/requires-config.yaml
 SEDFILE=${OUTPUT_DIR}/helm/sedfile
 export IMAGE_TAG
-export HELM_REGISTRY
+export HELM_REGISTRY=$(aws ecr-public describe-registries --region us-east-1  --output text --query 'registries[*].registryUri')
```

Comment:
It feels a little strange to me that we're specifying a registry, but not pushing to it if the push to a helm registry that configure via script succeeds. Is that really the intended behavior? How does the user know which registry received the push?

Reply:
There are several steps in a helm build. The push comes later. All the steps are run with a `make helm/push`